hooks/post/10-post_install.rb

```diff
@@ -52,7 +52,7 @@ def new_install?
   Once this is completed run the steps below to start the smartproxy installation:
 
   1. Ensure that the foreman-installer-katello package is installed on the system.
-  2. Copy <%= color("#{certs_tar}", :info) %> to the system <%= color("#{foreman_proxy_fqdn}", :info) %>
+  2. Copy the following file <%= color("#{certs_tar}", :info) %> to the system <%= color("#{foreman_proxy_fqdn}", :info) %> at the following location <%= color("#{File.join('/root', File.basename(certs_tar))}", :info) %>
```

Comment:
I'm not sure I like the ```the following file``` addition. Other than that you can add a suggestion: ```scp SOURCE PROXY:DEST```. While it may not work on every system, it does clearly show the intent.